CHANGELOG.md

```diff
@@ -2,6 +2,14 @@
 
 All notable changes to the Zowe API Mediation Layer package will be documented in this file.
 
+## `APIML 2.18.1 / Zowe 2.18.0 (2024-08-26)`
+
+* Feature:  Validation of OIDC token via user info endpoint (v2) (#3670) ([9fc5976](https://github.com/zowe/api-layer/commit/9fc5976)), closes [#3670](https://github.com/zowe/api-layer/issues/3670)
+
+* Bugfix:  enable optional authentication requirement for application/health endpoints (#3636) ([b41b071](https://github.com/zowe/api-layer/commit/b41b071)), closes [#3636](https://github.com/zowe/api-layer/issues/3636)
+* Bugfix:  support customized code snippets in case of endpoint with query params (#3665) ([5e024f4](https://github.com/zowe/api-layer/commit/5e024f4)), closes [#3665](https://github.com/zowe/api-layer/issues/3665)
```

Comment:
```suggestion
* Bugfix:  Support customized code snippets in case of endpoint with query params (#3665) ([5e024f4](https://github.com/zowe/api-layer/commit/5e024f4)), closes [#3665](https://github.com/zowe/api-layer/issues/3665)
```